make/conf/jib-profiles.js

```diff
@@ -453,7 +453,7 @@ var getJibProfilesProfiles = function (input, common, data) {
         "macosx-aarch64": {
             target_os: "macosx",
             target_cpu: "aarch64",
-            dependencies: ["devkit", "gtest", "graphviz", "pandoc"],
+            dependencies: ["devkit", "gtest", "graphviz", "pandoc", "tidy"],
```

Comment:
Is there a reason for not providing Tidy on macosx-x64? It looks like the binary built by the script below would be a multi-arch variant. If so, you should deploy it as just "tidy-html-macosx" and add some code in the dependencies section below that defines "module" as just `"tidy-html" + input.target_os` on macosx.